integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/AllDataTypesTestCaseAggregate.scala

```diff
@@ -59,21 +59,4 @@ class AllDataTypesTestCaseAggregate extends QueryTest with BeforeAndAfterAll {
       Seq(Row(100005.8)))
   })
 
-  test("CARBONDATA-60-union-defect")({
```

Comment:
here, only one test case is removed from carbon-spark module, but in test report, it says two are deleted, can you check why?
http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/560/testReport/